JabMapTestData/dummydata-1.bib

```diff
@@ -0,0 +1,63 @@
+% Encoding: UTF-8
+@INPROCEEDINGS{aPaper,
+	author={T. {Kim} and S. {Park} and T. {Lee}},
+	booktitle={2017 IEEE International Symposium on Software Reliability Engineering Workshops (ISSREW)}, 
+	title={Applying software reliability engineering process to software development in Korea defense industry}, 
+	year={2017},
+	volume={},
+	number={},
+	pages={81-81},
+}
+
+@INBOOK{aBook,
+	author={D. {Galin}},
+	booktitle={Software Quality: Concepts and Practice}, 
+	title={Software Testing}, 
+	year={2018},
+	volume={},
+	number={},
+	pages={255-317},
+}
+
+@MindMapNode{1,
+	id = {1},
+	name = {a node},
+	bibentry = {aBook},
+	icons = {to_read},
+	x_pos = {2},
+	y_pos = {2},
+}
+
+@MindMapNode{2,
+	id = {2},
+	name = {test number 2},
+	bibEntry = {aPaper},
+	icons = {read,low_priority},
+	x_pos = {3},
+	y_pos = {3},
+}
+
+@MindMapNode{3,
+	id = {3},
```

Comment:
Ah, this really is work-in-progress - we discussed to remove `id` (as this is the BibTeX key)

Reply:
Was unaware of the changes to the fields at the time. Data has now been updated to the current version.